hpe_3par_kubernetes_manager.py

```diff
@@ -310,12 +352,39 @@ def hpe_list_secret_objects_names(namespace):
         raise e
 
 
+def hpe_list_service_objects(namespace):
+    try:
+        service_list = k8s_core_v1.list_namespaced_service(namespace=namespace)

```

Comment:
Are you sure you will always get a service list? How can you ensure that the service list won't be empty?

Reply:
Service list can be empty, but we need not ensure it won't be empty as we are getting the list of services under a namespace that are currently created. If it is empty, service_list will be an empty list.